### PR TITLE
Issue #822: Introduction of YML metadata for eclipse-cs integration

### PR DESCRIPTION
**Pull request by gaurabdg**
#822 
Details are mentioned in the issue

### CODE REVIEW

File: eclipsecs-sevntu-plugin/src/com/github/sevntu/eclipse-metadata.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+extensionName: "com.github.sevntu"
+ruleGroupName: "SevNTU checks"
+ruleGroupDesc: "Check extensions developed by SevNTU"
```
Comment (romani) [inline eclipsecs-sevntu-plugin/src/com/github/sevntu/eclipse-metadata.yml:4]: In original config there is some number also  that might be used to define position of extension in tree of modules in eclipse-cs UI

Comment (romani) [reply eclipsecs-sevntu-plugin/src/com/github/sevntu/eclipse-metadata.yml:4]: Where we can define `priority`
https://github.com/sevntu-checkstyle/sevntu.checkstyle/blob/83377c1c1d25a95f36837d4b3fa6e721d108a8aa/eclipsecs-sevntu-plugin/src/com/github/sevntu/checkstyle/checks/annotation/checkstyle-metadata.xml#L6

?

Comment (gaurabdg) [reply eclipsecs-sevntu-plugin/src/com/github/sevntu/eclipse-metadata.yml:4]: @romani  I will add a field for it.
But why is `priority` different in some packages whereas rule group is same?
https://github.com/sevntu-checkstyle/sevntu.checkstyle/blob/83377c1c1d25a95f36837d4b3fa6e721d108a8aa/eclipsecs-sevntu-plugin/src/com/github/sevntu/checkstyle/checks/coding/checkstyle-metadata.xml#L6

Comment (romani) [reply eclipsecs-sevntu-plugin/src/com/github/sevntu/eclipse-metadata.yml:4]: @Gaurav-Punjabi , it is mistake, it should be the same.

Comment (gaurabdg) [reply eclipsecs-sevntu-plugin/src/com/github/sevntu/eclipse-metadata.yml:4]: > Where we can define `priority`
> https://github.com/sevntu-checkstyle/sevntu.checkstyle/blob/83377c1c1d25a95f36837d4b3fa6e721d108a8aa/eclipsecs-sevntu-plugin/src/com/github/sevntu/checkstyle/checks/annotation/checkstyle-metadata.xml#L6
> 
> ?

@romani Done.

Comment (romani) [reply eclipsecs-sevntu-plugin/src/com/github/sevntu/eclipse-metadata.yml:4]: We should give ability to use multiple groups from single extension in yml, please do

Comment (gaurabdg) [reply eclipsecs-sevntu-plugin/src/com/github/sevntu/eclipse-metadata.yml:4]: I didnt understand why would we need multiple groups, please provide and example/use case

Comment (gaurabdg) [reply eclipsecs-sevntu-plugin/src/com/github/sevntu/eclipse-metadata.yml]: > We should give ability to use multiple groups from single extension in yml, please do

@romani Done. The explanation is present in the issue description

**Comment by gaurabdg** [comment]
> ```
> ruleGroups:
>   - 
>     name: "SevNTU checks"
>     description: "Check extensions developed by SevNTU"
>     priority: 50
>     package: "com.github.sevntu"
> ```

Done

**Pull request merged**